### PR TITLE
Documentation: Minor corrections of several md files

### PR DESCRIPTION
**Pull request by MarchMore**

### CODE REVIEW

File: docs/project/evolution.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ This process is designed to:
 
 -   Ensure these kinds of changes can receive feedback from the entire
     community.
--   Resolve questions and decide direction efficiently.
+-   Resolve questions and decide directions efficiently.
```
Comment (jonmeow) [inline docs/project/evolution.md]: direction here should remain singular -- i.e., we're taking Carbon on one route, not multiple.

File: docs/project/goals.md [hunk 2]
```diff
@@ -76,7 +76,7 @@ community to be welcoming and respectful, with a deep commitment to
 psychological safety. We need consistent expectations for how every community
 member should behave, regardless of their position in the community. These
 expectations around conduct and behavior need to be clearly articulated both to
-set expectations for people joining, and to help remind and anchor us on
+set expectations for people joining, and to help remind and anchor us to
```
Comment (jonmeow) [inline docs/project/goals.md]: I think "on" was fine here.

File: docs/project/roadmap.md [hunk 3]
```diff
@@ -112,7 +112,7 @@ both measures our progress solidifying Carbon's design and demonstrating the
 value proposition of Carbon.
 
 Note that both woff2 and RE2 libraries are chosen somewhat arbitrarily and could
-easily be replaced with a different, more effective libraries to achieve the
+easily be replaced with a different, more effective library to achieve the
```
Comment (jonmeow) [inline docs/project/roadmap.md]: I think plural libraries was correct here.

File: docs/project/roadmap.md [hunk 4]
```diff
@@ -155,7 +155,7 @@ Stretch goals if we can hit the above:
     integrated.
 
 Benchmarking at this stage isn't expected to include extensive optimization.
-Instead, it should focus on letting us track large/high-level impact on
+Instead, it should focus on letting us track large/high-level impacts on
```
Comment (jonmeow) [inline docs/project/roadmap.md]: I think singular impact was fine here.

**Review by jonmeow — COMMENTED** [review]
While some of these corrections are objectively good, I think several of these aren't objective grammar choices. In these cases we'll often prefer to stick with previous phrasing, in order to minimize churn.

**Comment by MarchMore** [comment]
@jonmeow, I fully understand. I reverted the affected parts

File: CODE_OF_CONDUCT.md [hunk 5]
```diff
@@ -294,7 +294,7 @@ involved:
 
 1.  **Correction**
     -   **Behavior:** Use of inappropriate language or other minor violations
-        the code of conduct.
+        of the code of conduct.
```
Comment (jonmeow) [inline CODE_OF_CONDUCT.md]: This "of" needs to be on the previous line (we use prettier for auto-formatting):
https://github.com/carbon-language/carbon-lang/runs/7568551514?check_suite_focus=true

Comment (MarchMore) [reply CODE_OF_CONDUCT.md]: Fixed

**Review by jonmeow — APPROVED** [review]
Thanks!

**Pull request merged**